specification/monitor/resource-manager/Microsoft.Insights/preview/2023-06-01-preview/privateLinkScopes_API.json

```diff
@@ -0,0 +1,1110 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MonitorManagementClient",
+    "x-ms-code-generation-settings": {
+      "name": "MonitorManagementClient"
+    },
+    "description": "Azure Monitor API reference for Private Links Scopes management.",
+    "version": "2023-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Insights/privateLinkScopes": {
```

Comment:
>Microsoft.Insights/ [](http://example.com/codeflow?start=47&length=19)

While I think this is added to fix a linter error...

I seem to remember this (Change casing of RP namespace in swagger) can also detect as a breaking change, since URIs are case sensitive... even though provider names are mainly supposed to be processed insensitively. @JeffreyRichter any real known problems with changing this?

Reply:
For ARM/RPs, changing URL casing is not considered a breaking change so it is fine here.
(For data plane, it is breaking.)

Reply:
Great, thanks for clearing that up!